src/components/Layout/Boxes.jsx

```diff
@@ -5,7 +5,7 @@ import { IconWarn, IconError } from '../Icons'
 /**
  * Box base styling
  */
-export const BoxBase = styled.div`
+const BoxBase = styled.div`
```

Comment:
We _could_, but should we create there the possibility for custom boxes in other apps? Afraid it could get messy, currently there is no need